readme.md

```diff
@@ -50,17 +50,18 @@ Usage of ./nitro-rpc-server:
   -usenats
         Specifies whether to use NATS or http/ws for the rpc server.
 ```
-  
+
 You can make remote procedure calls like so:
-  
-  ```shell
-  curl -X POST \   
-     -H 'Content-Type: application/json' \
-     -d '{"jsonrpc":"2.0","id":1,"method":"get_address","params":{}}' \
-     http://localhost:4005/api
-  ```
-
-  but see https://github.com/statechannels/nitro-gui for an RPC client to do so programmatically. 
+
+```shell
+curl -X POST \
+  -H 'Content-Type: application/json' \
+  -d '{"jsonrpc":"2.0","id":1,"method":"get_address","params":{}}' \
+  http://localhost:4005/api/v1
+```
```

Comment:
Not sure why but reformatting the curl command above allowed me to do a copy+paste into the terminal and run the command. The original format produced an error with the curl command